### PR TITLE
[Bugfix] Fix GAT on PPI

### PR DESCRIPTION
**Pull request by mufeili**
## Description
As reported in [thread 979](https://discuss.dgl.ai/t/bug-reported-the-evaluation-function-of-gat-ppi/979), previously the f1 evaluation for PPI is wrong. Since sigmoid has not been applied to the model output, the output values are logits rather than probabilities and the threshold for binary classification should be 0 rather than 0.5. Recall that sigmoid takes the form 1 / 1 + e^{-z}.

## Checklist
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented
- [x] To the my best knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [x] Related issue is referred in this PR

### CODE REVIEW

**Comment by ycjing** [comment]
Hi @mufeili 
Thank you for the quick response and also quick fix!

Best,
Yongcheng
